rakefile-dotnet.rb

```diff
@@ -258,6 +268,12 @@
   cmd.parameters "pack #{nuspec} -Version #{version}#{version_suffix} -OutputDirectory #{output}"
 end
 
+desc "create the nuget package with the .NET Standard"
+exec :pack => [:build, output] do |cmd|
+  cmd.command = nuget_command
+  cmd.parameters "pack #{dotnet_nuspec} -Version #{version}-dotnet#{version_suffix} -OutputDirectory #{output}"
+end
+

```

Comment:
> We'll need a separate env var on the CI server to control the pre-release moniker for this rakefile though, perhaps VERSION_SUFFIX_DOTNET.

not `_NETSTD`?

But seriously, why? We have the ability to hardcode such a thing into the version number right now (as I do on line 274 above). We won't want to remove the special dotnet/netstd token until we promote this rakefile to be _the_ rakefile, so why not just leave it in here?

> More generally, I don't think the new rakefile should care at all about the net40-only situation. It should only be concerned with doing everything required for producing a net40+netstandard1.6 package.

Okay.